/**:
  ros__parameters:
    dlo:
      version: 1.4.3
      adaptiveParams: true
      imu: true
      gravityAlign: true

      odomNode:
        odom_frame: odom
        child_frame: base_link
        publishTF: true

      mapNode:
        publishFullMap: true
        publishFreq: 1.0
        leafSize: 0.25
